lms/views/dashboard.py

```diff
@@ -79,3 +68,28 @@ def api_assignment_stats(self):
             }
             for s in stats
         ]
+
+    def get_request_assignment(self):
+        assignment = self.assignment_service.get_by_id(self.request.matchdict["id_"])
+        if not assignment:
+            raise HTTPNotFound()
+
+        if not self.assignment_service.is_member(assignment, self.request.user):
```

Comment:
We trust the ID from the URL, not the one from LTIUser.

We must check that we do indeed have access to URL assignment.